src/CMetricsCalculator.cpp

```diff
@@ -766,10 +793,12 @@ CMetricsCalculator::calculate_metrics_switch()
 			break;
 		case CKeyword::STRUCT:
 			qm.add_struct();
+            saw_struct = true;
```

Comment:
Please fix indentation.  (Ensure your editor's settings match the code's style.)